Documentation/gettingstarted/minikube_intro.rst

```diff
@@ -7,16 +7,19 @@ Step 0: Install kubectl & minikube
 
 3. Install ``minikube`` ``>= 0.22.3`` as described on `minikube's github page <https://github.com/kubernetes/minikube/releases>`_ .
 
+.. tabs::
+  .. group-tab:: docker
 
-Boot a minikube cluster with the Container Network Interface (CNI) network
-plugin, the ``localkube`` bootstrapper.
+    .. parsed-literal::
 
-The ``localkube`` bootstrapper provides ``etcd`` >= ``3.1.0``, a cilium
-dependency.
+      $ minikube start --network-plugin=cni --extra-config=kubelet.network-plugin=cni --memory=5120
```

Comment:
We no longer specify 'localkube' - does this mean we will work with any bootstrapper? Or should we specify which one is preferred still?

Reply:
localkube is deprecated.

Reply:
OK, I looked online and kubeadm is the only other bootstrapper.